client/js/apollo/index.js

```diff
@@ -0,0 +1,34 @@
+import {ApolloClient} from 'apollo-client';
+import {ApolloLink} from 'apollo-link';
+import {createHttpLink} from 'apollo-link-http';
+import {InMemoryCache} from 'apollo-cache-inmemory';
+import {onError} from 'apollo-link-error';
+
+const httpLink = createHttpLink({
+  includeExtensions: true,
+  uri:
+    process.env.NODE_ENV !== 'production' &&
+    'https://us1.prisma.sh/public-flintwanderer-148/server/dev',
+  credentials:
+    process.env.NODE_ENV === 'production' ? 'same-origin' : 'include',
+});
+
+const client = new ApolloClient({
+  link: ApolloLink.from([
+    onError(({graphQLErrors, networkError}) => {
+      // Log better error messages to console
+      if (graphQLErrors) {
+        graphQLErrors.map(({message, locations, path}) =>
+          console.log(
+            `[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`,
+          ),
+        );
+      }
+      if (networkError) console.log(`[Network error]: ${networkError}`);
+    }),
+    httpLink,
+  ]),
+  cache: new InMemoryCache(), // Pull data from client-side cache, if available
+});
```

Comment:
Any reason to change your Apollo client now? Your app is not in the production mode and you are not storing cookies on the client so no need for `process.env.NODE_ENV === 'production' ? 'same-origin' : 'include',`.

Please use Apollo boost.

Reply:
Deleted